habitat-lab/habitat/config/benchmark/rearrange/cat_pick.yaml

```diff
@@ -29,9 +29,21 @@ habitat:
     agents:
       main_agent:
         radius: 0.3
```

Comment:
How are you handeling the different agent radius? Are you regenerating the navmesh for Stretch? I assume the radius of the two robots is similar, but we should be precise since this will affect everything.

Reply:
Currently, we are using the navmesh that was generated with 0.2 radius. We will regenerate navmesh for all scenes with the correct Stretch radius and add them to the scenes directory.

Reply:
Lets still use the correct radius here and in cat_place to prevent confusion for others.

Reply:
Update to 0.25